handbook/company/pricing-features-table.yml

```
@@ -74,50 +269,123 @@
       moreInfoUrl: https://github.com/fleetdm/fleet/issues/19037
     - description: Require end users to authenticate with your identity provider (IdP) and agree to an end user license agreement (EULA) before they can use their new workstation
 #  
+#  ╔╗ ╦ ╦╔═╗╔╦╗  ╔═╗╔╗╔╦═╗╔═╗╦  ╦  ╔╦╗╔═╗╔╗╔╔╦╗
+#  ╠╩╗╚╦╝║ ║ ║║  ║╣ ║║║╠╦╝║ ║║  ║  ║║║║╣ ║║║ ║ 
+#  ╚═╝ ╩ ╚═╝═╩╝  ╚═╝╝╚╝╩╚═╚═╝╩═╝╩═╝╩ ╩╚═╝╝╚╝ ╩ 
+- industryName: Bring Your Own Device (BYOD) enrollment
```

```suggestion
- industryName: Bring your own device (BYOD) enrollment
```

Sentence casing here too I think.